homeassistant/components/loxone.py

```diff
@@ -0,0 +1,230 @@
+"""
+Component to create an interface to the Loxone Miniserver
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/loxone/
+"""
+# pylint: disable=unused-import, too-many-lines
+import asyncio
+import codecs
+import hashlib
+import hmac
+import logging
+import uuid
+import json
+from datetime import timedelta
+from struct import unpack
+
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (CONF_HOST, CONF_PORT, CONF_USERNAME,
+                                 CONF_PASSWORD)
+from homeassistant.helpers.event import async_track_time_interval
+
+REQUIREMENTS = ['websockets==3.4']
```

Comment:
You need to update the requirement of the other platform that uses `websockets` as well.

Reply:
I think my biggest problem is that I do not know how to pull the latest changes from dev. 
I tried it without a gui for github but I did not get it to work. Now I tried GitKraken. I fixed all of 
the pylint issues. Maybe it is better you delete my pull request and I delete my fork. And I first try to get used with github and git.

Reply:
And also I have big problems with the documentation page. But this is also a problem because I do not understand the github way to do it. I work personally with subversion and this is much easier for me. I believe that git is better but also more complicated.